deebotozmo/cli.py

```diff
@@ -0,0 +1,439 @@
+        print("Stats Cid: " + str(event.clean_id))
+        print("Stats Area: " + str(event.area))
+        if isinstance(event.time, int):
+            print("Stats Time: " + str(int(event.time / 60)) + " minutes")
+        print("Stats Type: " + str(event.type))
+        lock.release()
+
+    listener = vacbot.bot.events.stats.subscribe(on_stats_event)
+    await lock.acquire()
+    vacbot.bot.events.stats.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(help="Get robot components life span")
+@coro
+async def components() -> None:
+    """Click subcommand that returns the robot's life span."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_lifespan_event(event: dict) -> None:
+        for key, value in event.items():
+            print(f"{key}: {value}%")
+        lock.release()
+
+    listener = vacbot.bot.events.lifespan.subscribe(on_lifespan_event)
+    await lock.acquire()
+    vacbot.bot.events.lifespan.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(name="getrooms", help="Get saved rooms")
+@coro
+async def get_rooms() -> None:
+    """Click subcommand that returns saved room."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_rooms(event: RoomsEvent) -> None:
+        for room in event.rooms:
+            print(str(room.id) + " " + str(room.subtype))
+        lock.release()
+
+    listener = vacbot.bot.map.events.rooms.subscribe(on_rooms)
+    await lock.acquire()
+    vacbot.bot.map.events.rooms.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(
+    name="exportlivemap",
+    help='Get robot map and save it [filepath ex: "/folder/livemap.png"',
+)
+@click.argument("filepath", type=click.STRING, required=True)
+@coro
+async def export_live_map(filepath: str) -> None:
+    """Click subcommand that returns the live map."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_map(_: MapEvent) -> None:
+        with open(filepath, "wb") as file:
+            file.write(base64.decodebytes(vacbot.bot.map.get_base64_map()))
+        lock.release()
+
+    listener = vacbot.bot.events.map.subscribe(on_map)
+    await lock.acquire()
+    vacbot.bot.events.map.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+class DoLogin:
+    """DoLogin communicates with the bot to provide self.bot."""
+
+    def __init__(self) -> None:
+        """Init DoLogin."""
+        # self.session = None
+        # self.auth = None
+        # self.devices_ = None
+        # self.bot = None
+        # self.mqtt = None
+        return
+
+    async def run(self) -> None:
+        # pylint: disable=attribute-defined-outside-init
+        """Communicate with Deebot."""
+        config = read_config()
+
+        self.session = aiohttp.ClientSession()
```

Comment:
```suggestion
        self._session = aiohttp.ClientSession()
```

Variables not required outside the class should be prefixed with an `_` (mark the as "protected")